assets/helpers/contributions.js

```diff
@@ -80,6 +80,12 @@ type Config = {
   }
 }
 
+export type OtherAmounts = {
+  [ContributionType]: { amount: string | null }
```

Comment:
not really your PR but I wonder whether this `amount` could be of type `Amount`? or maybe better, rename type `Amount` to `PresetAmount` since that's really what it's modelling